Carbon.Security/Functions/Get-CPrivilege.ps1

```diff
@@ -0,0 +1,64 @@
+
+function Get-CPrivilege
+{
+    <#
+    .SYNOPSIS
+    Gets an account's rights and privileges.
+
+    .DESCRIPTION
+    The `Get-CPrivilege` function gets an account's rights and privileges. These privileges are usually managed by Group
+    Policy and control the system operations and types of logons an account can perform.
+
+    Note: if a computer is not on a domain, this function won't work.

```

Comment:
Is this still the case? Won't work how? Writes an error?

Reply:
I have no idea. This is original documentation so have no idea. I don't think it's accurate, though, since our builds run on computers not on domains.